### PR TITLE
build_testapps.py tvOS device profiles

### PR DESCRIPTION
**Pull request by DellaBitta**
* Update `build_testapps.py` to pass the target os (`iOS` or `tvOS`) to the `xcodebuild.py` build script.
* `xcodebuild.py` now populates the correct device profiles `appletvos` and `appletvsimulator` for `device` and `simulator` iOS SDK types, respectively.

***

- Building tvOS testapps locally with `build_testapps.py` and in editor.
- Building iOS testapps locally to esnure that logic hasn't regressed.
- [Integration Test CI](https://github.com/firebase/firebase-unity-sdk/actions/workflows/integration_tests.yml)

***

### Type of Change
Place an `x` the applicable box:
- [ ] Bug fix. Add the issue # below if applicable.
- [ ] New feature. A non-breaking change which adds functionality.
- [X] Other, such as a build process or documentation change.
***

### CODE REVIEW

File: scripts/gha/integration_testing/xcodebuild.py [hunk 1]
```diff
@@ -79,14 +80,24 @@ def get_args_for_build(path, scheme, output_dir, ios_sdk, configuration):
...
+  elif target_os == "tvOS":
+    if ios_sdk == "device":
+      return "appletvos"
+    elif ios_sdk == "simulator":
+      return "appletvsimulator"
+    else:
+      raise ValueError("Unrecognized tvOS ios_sdk parameter: %s" % sdk)
   else:
-    raise ValueError("Unrecognized ios_sdk: %s" % ios_sdk)
+    raise ValueError("Unrecognized target_os %s" % target_os)
```
Comment (cynthiajoan) [inline scripts/gha/integration_testing/xcodebuild.py:100]: nit: probably also worth to log what's the ios_sdk here.

File: scripts/gha/integration_testing/xcodebuild.py [hunk 2]
```diff
@@ -46,6 +46,7 @@ def get_args_for_build(path, scheme, output_dir, ios_sdk, configuration):
     output_dir (str): Directory for the resulting build artifacts. Will be
         created if it doesn't already exist.
     ios_sdk (str): Where this build will be run: device or simulator.
```
Comment (cynthiajoan) [inline scripts/gha/integration_testing/xcodebuild.py:48]: nit: to add "" to device / simulator to keep consistency with the newly added line :)

**Review by cynthiajoan — APPROVED** [review]

File: scripts/gha/integration_testing/xcodebuild.py (hunk 1 above)
Comment (DellaBitta) [reply scripts/gha/integration_testing/xcodebuild.py:100]: Good idea. Done!

File: scripts/gha/integration_testing/xcodebuild.py (hunk 2 above)
Comment (DellaBitta) [reply scripts/gha/integration_testing/xcodebuild.py:48]: Done!

**Review by a-maurice — APPROVED** [review]

**Pull request merged**